industry_benchmarks/utils/data_gathering.py

```diff
@@ -344,10 +344,31 @@ def get_fingerprint_similarity_score(mapping: LigandAtomMapping) -> float:
     fpA = fpgen_morgan.GetFingerprint(molA)
     fpB = fpgen_morgan.GetFingerprint(molB)
     # Get Tanimoto similarity
-    morgan_fp_score = rdkit.DataStructs.TanimotoSimilarity(fpA, fpB)
+    morgan_fp_score = DataStructs.TanimotoSimilarity(fpA, fpB)
 
     return morgan_fp_score
 
+def get_atom_pair_similarity(mapping: LigandAtomMapping) -> float:
+    molA = mapping.componentA.to_rdkit()
+    molB = mapping.componentB.to_rdkit()
+    Chem.SanitizeMol(molA)
+    Chem.SanitizeMol(molB)
+
+    fp_a = rdMolDescriptors.GetHashedAtomPairFingerprintAsBitVect(molA)
+    fp_b = rdMolDescriptors.GetHashedAtomPairFingerprintAsBitVect(molB)
+    return DataStructs.TanimotoSimilarity(fp_a, fp_b)
```

Comment:
If we look at the rdkit docs and also the paper you linked me (https://pmc.ncbi.nlm.nih.gov/articles/PMC4932683/#CR26), I believe DiceSimilarity is the proposed metric that folks usually use for atom pair and topological torsion.

That being said I definitely am no expert in this, is TanimotoSimilarity intended?